openedx/core/djangoapps/edx_global_analytics/tasks.py

```diff
@@ -135,4 +139,9 @@ def count_data():
             'students_per_country': json.dumps(students_per_country)
         })
 
-    requests.post(post_url, data)
+    try:
+        request = requests.post(post_url, data)
+        logger.info('Connected without error to {0}'.format(request.url))
+        logger.info('Data was successfully sent. Status code is {0}.'.format(request.status_code))
+    except requests.ConnectionError as error:
```

Comment:
please read http://docs.python-requests.org/en/latest/user/quickstart/#errors-and-exceptions
```
In the event of a network problem (e.g. DNS failure, refused connection, etc), Requests will raise a ConnectionError exception.
In the event of the rare invalid HTTP response, Requests will raise an HTTPError exception.
If a request times out, a Timeout exception is raised.
If a request exceeds the configured number of maximum redirections, a TooManyRedirects exception is raised.
All exceptions that Requests explicitly raises inherit from requests.exceptions.RequestException.
```
You can either catch the base-class exception `RequestException `, which will handle all cases or you can catch them separately and do different things